ticdc/ticdc-sink-to-kafka.md

```diff
@@ -36,13 +36,13 @@ Info: {"sink-uri":"kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-v
 Sink URI is used to specify the connection information of the TiCDC target system. The format is as follows:
 
 ```shell
-[scheme]://[userinfo@][host]:[port][/path]?[query_parameters]
+[scheme]://[host]:[port],[host]:[port],[host]:[port][/path]?[query_parameters]
```

Comment:
```suggestion
[scheme]://[host]:[port][/path]?[query_parameters]
```

Reply:
we need to indicate `Sink URL` supports multiple hosts.